### DISCUSSION TITLE
Variable number of rows and colums.

### DISCUSSION THREAD

**Discussion by qualto** [VA3HDL/hamdashboard#30] [Ideas]
I would like to be able to configure a 2x2 grid instead of 4 x 3.  Others may wish to have other combinations, thus the proposal to make that configurable. 

I have forked and modified this for my needs to display 2x2 grid. https://github.com/qualto/hamdashboard

**Comment by technoweenie1** [comment]
I second this idea! What did you modify to get a different grid arrangement? I would like a 2 high by 3 wide.

**Reply by qualto** [reply]
config.js
1. Removed the "excess" dashboard items. in your case leave 6.
2. Removed the "excess" image rotation intervals at the bottom of the file.   Again for you case leave 6. 

hamdash.html
1. Line 123.   "grid-template-columns: repeat(4, 2fr);"  change from 'repeat(4, 2fr)' to 'repeat(3, 2fr)'
2. Lines 138 and 139  set the image scaling. you may need to tinker with these settings to get the image to scale properly. 
        width: 49vw;  <-- Change the number here to adjust the image widths
        height: 46vh; <-- Change the number here to adjust the image heights
4. Lines 327 and 328 Delete the excess "1"s and "null"s
` var aIdx = [1, 1, 1, 1, 1, 1];
   var aInt = [null,null,null,null,null,null];`

If someone has a better way to accomplish this please feel free to chime in.

**Comment by technoweenie1** [comment]
Worked like a charm. Thanks!
For me on a 3x2 grid, the horiz and vert are:
width: 33.5vw;
height: 46.5vh;

    Bryan VE3OYN.

**Comment by VA3HDL** [comment]
Hello,
Now this functionality is built-in the dashboard. Please try the new version and let me know if any issues.
Thanks.
